.github/workflows/build-push.yml

```diff
@@ -0,0 +1,40 @@
+name: Build + Push to Azure
```

Comment:
Can we add "docs" to this name (and the name of the file itself) so it's clear what this is for? (and not the product).